package.json

```diff
@@ -17,9 +17,13 @@
   },
   "dependencies": {
     "code-push": "1.5.2-beta",
-    "semver": "^5.1.0"
+    "semver": "^5.1.0",
+    "rnpm": "1.4.1"
   },
-  "devDependencies": {
+  "peerDependencies": {
     "react-native": "0.19.0"
+  },
+  "scripts": { 
+    "postinstall": "cd ../../ && rnpm link react-native-code-push"
```

Comment:
Is it really necessary to cd up like this? Did you get this solution from the RNPM team? It just kind of seems strange. I would have expected the RNPM script to walk up the CWD until it found an RN app or something similar.